pkgs/applications/virtualization/docker/containerd.nix

```diff
@@ -0,0 +1,57 @@
+# this is pinned because it still uses the old buildGoPackage
```

Comment:
Also why do we have this file now doubled?

Reply:
the override logic in docker assumes it still uses the `buildGoPackage` helper, not the `buildGoModule`